packages/sources/ion.au/src/transport/total_reserve.ts

```diff
@@ -0,0 +1,66 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { BaseEndpointTypes } from '../endpoint/total_reserve'
+
+export interface ResponseSchema {
+  total_reserve: number
+  errorMessage?: string
+}
+
+export type HttpTransportTypes = BaseEndpointTypes & {
+  Provider: {
+    RequestBody: never
+    ResponseBody: ResponseSchema
+  }
+}
+export const httpTransport = new HttpTransport<HttpTransportTypes>({
+  prepareRequests: (params, config) => {
+    return params.map((param) => {
+      return {
+        params: [param],
+        request: {
+          baseURL: config.API_ENDPOINT,
+          url: '/',
+          headers: {},
+          params: {},
+        },
+      }
+    })
+  },
+  parseResponse: (params, response) => {
+    if (response.data.errorMessage) {
+      return params.map((param) => {
+        return {
+          params: param,
+          response: {
+            errorMessage: `There was an error from the source API`,
```

Comment:
Could you also append the `response.data.errorMessage` here. It'll help with debugging if an error ever does come up
```suggestion
            errorMessage: `There was an error from the source API. ${response.data.errorMessage}`,
```